learn/wavemaker-release-notes/v10-6-7.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "WaveMaker 10.6.7 - Release date: 11 March 2021"
+id: ""
+sidebar_label: "Release v10.6.7"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Improvements
+
+---
+
+- We introduced a new property called **Show Display Format On** for the Text widget to apply a defined **Display Format** property when entering text.
+- We introduced the **On Clear** event for the Search widget to clear the field.
+
+## Bug Fixes
+
+---
+
+- We fixed an issue related to logout action failing with NPE (Null Pointer Exception) when using SSO OpenID with Horizontal Scaling enabled.
+- We fixed an issue related to the **Match Mode** property in the Data table form, forming an altered FilterExperssion for autocomplete, select, and search type.
+- We fixed an issue related to values modifying from negative to positive automatically when specified in the Number widget.
+- We fixed an issue related to the browser's autocomplete suggestion even when the **Autocomplete** property is turned off.
+- We fixed an issue related to the **On Change** event of the Number widget. Precisely, when a number widget is inside a Live form, it automatically triggers the **On Change** event when there are no actual changes in the event value.
```

Comment:
"Precisely, when a number widget was inside a Live form, it used to automatically trigger the **On Change** event even when there were no actual changes in the number value."